app/controllers/static_controller.rb

```diff
@@ -0,0 +1,8 @@
+class StaticController < ApplicationController
+  def index; end
+
+  def greetings
+    @greetings = Greeting.order(Arel.sql('RANDOM()')).limit(5)
```

Comment:
- [ ] Your application returns multiple greetings, whereas you are expected to return only one. Kindly fix that.

- [ ] In addition to that kindly streamline your API response to return only the `text` attribute. It's not particularly a good practice to return all active record attributes during an API call

  ![response](https://user-images.githubusercontent.com/54848363/183670955-3c5e1b12-3676-4d20-b10a-14768c440204.jpg)